libs/react-helpers/src/lib/grid/cumulative-vol-cell.tsx

```diff
@@ -13,7 +13,7 @@ export interface CumulativeVolProps {
   className?: string;
 }
 
-export interface ICumulativeVolCellProps extends ICellRendererParams {
+export interface AgCumulativeVolCellProps extends ICellRendererParams {
```

Comment:
chore (blocking): I think we are crossing concerns here. We need to differentiate what is used in an ag grid and what is not. This particular component is only used in the order book.

Can we
a) not rename it to AgGridCumulative...
b) co locate it with the order book